src/components/Nav/index.tsx

```diff
@@ -372,23 +304,43 @@ const Nav: FC<IProps> = ({ path }) => {
       ? `?from=/${splitPath.slice(2).join("/")}`
       : ""
   return (
-    <NavContainer>
-      <StyledNav aria-label={t("nav-primary")}>
-        <NavContent>
-          <HomeLogoNavLink to="/" aria-label={t("home")}>
-            <HomeLogo />
-          </HomeLogoNavLink>
+    <Box position="sticky" top={0} zIndex={100} width="full">
+      <Flex
+        as="nav"
+        aria-label={t("nav-primary")}
+        bg="background"
+        borderBottom="1px"
+        borderColor="rgba(0, 0, 0, 0.1)"
+        height="4.75rem"
+        justifyContent="center"
+        py={4}
+        px={{ base: 8, lg: 4, xl: 8 }}
+      >
+        <Flex
+          alignItems={{ base: "center", lg: "normal" }}
+          justifyContent={{ base: "space-between", lg: "normal" }}
+          width="full"
+          maxW="1440px"
```

Comment:
@pettinarip would you like to go ahead and add this token value to the theme config?

Reply:
@pettinarip there is currently no `container.2xl` in the [default chakra config](https://github.com/chakra-ui/chakra-ui/blob/c14615106066349955b0eaa7b78d4f3a59a08b03/packages/components/theme/src/foundations/sizes.ts#L24-L29) so it would have to be added.

Reply:
Oh good to know! I thought all the breakpoints were defined there. Ok, will do!

Good to see that they are using `px` values in there instead of relative ones.